tests/phpunit/includes/install.php

```diff
@@ -37,6 +37,16 @@
 
 tests_add_filter( 'wp_die_handler', '_wp_die_handler_filter_exit' );
 
+// Ensure block view scripts are present on disk to avoid realpath() returning false in get_block_asset_url().
+foreach ( glob( ABSPATH . 'wp-includes/blocks/*/block.json' ) as $block_json ) {
```

Comment:
@westonruter we have this file- https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks/blocks-json.php